psignifit/_posterior.py

```diff
@@ -248,10 +248,11 @@ def objective(x):
 
     # "sorted" is required so that the parameters are matched to the axes in  alphabetical order
     init_values = [value for name, value in sorted(param_init.items())]
-    optimized_values = optimize.fmin(objective, init_values, disp=False)
+    optimized_values = [cast_np_scalar(value) for value in optimize.fmin(objective, init_values, disp=False)]
```

Comment:
This is correct but makes the line less readable, the line is important and its main intent is finding the MAP point estimate.
How about leaving it as it is, and at the end of the function declaring a `out_optimized_values` that does the casting, as in `out_param_fixed`? and they could be calles `cast_optimized_values` and `case_param_fixed`